apps/prairielearn/src/batched-migrations/20250212010945_variants__authn_user_id__backfill.ts

```diff
@@ -0,0 +1,21 @@
+import { makeBatchedMigration } from '@prairielearn/migrations';
+import { loadSqlEquiv, queryAsync } from '@prairielearn/postgres';
+
+const sql = loadSqlEquiv(import.meta.url);
+
+export default makeBatchedMigration({
+  async getParameters() {
+    const results = await queryAsync(
+      'SELECT MIN(id) AS min, MAX(id) AS max FROM variants WHERE authn_user_id IS NULL;',
```

Comment:
This hung when I tried it in prod, maybe we get rid of the `MIN`? It ran faster when just `MAX` was queried.

Reply:
I switched to a different query in cda4b42 that results in a better query plan.